### ISSUE TITLE
new features expected ...

### ISSUE THREAD

**Issue by colben** [svenstaro/miniserve#510]
Here are some new features expected ... this may sounds a little greedy and shameless as a free user like me ...
- upload directories
- create, rename, delete and move directories
- rename, delete and move files

**Comment by chaooe** [comment]
Yes, i also wanted if it can delete directories. and delete files

**Comment by svenstaro** [comment]
This should be mostly done except for rename.
